webpack.config.js

```diff
@@ -19,6 +19,8 @@ module.exports = {
     path: path.resolve(__dirname, "dist/client"),
     publicPath: "/client",
   },
+  stats: "minimal",
+  mode: "none",
```

Comment:
I think we can probably just not have this at all, and set the mode to development when we set up the webpack middleware (which is only done on dev mode).